src/dashboard/Pipfile

```
@@ -0,0 +1,15 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+django = "5.1.2"
+honcho = "==2.0.0"
+psycopg2-binary = "2.9.10"
```

Let's move to psycopg3!

```suggestion
psycopg = {extras = ["pool", "binary"], version = "==3.2.3"}
```